enums.py

```diff
@@ -108,6 +114,14 @@ class Tablenames(Enum):
     PERSONAL_ACCESS_TOKEN = "personal_access_token"
     ADMIN_MESSAGE = "admin_message"
     TASK_QUEUE = "task_queue"
+    CONVERSATION = "conversation"
+    MESSAGE = "message"
+    STRATEGY = "strategy"
+    STRATEGY_STEP = "strategy_step"
+    RETRIEVER = "retriever"
+    ENVIRONMENT_VARIABLE = "environment_variable"
+    PIPELINE_LOGS = "pipeline_logs"
+    MARKDOWN_FILE = "markdown_file"
```

Comment:
I suggest creating an own `Tablenames` enum for Cognition Tables

Reply:
especially as the same enum values are used for different tables (project, personal_access_token)